packages/editor-ui/src/components/Folders/FolderCard.vue

```diff
@@ -0,0 +1,217 @@
+		emit('folderOpened', { folder: props.data });
+		await router.push(cardUrl.value);
+		return;
+	}
+	emit('action', { action, folderId: props.data.id });
+};
+
+const onBreadcrumbsItemClick = async (item: PathItem) => {
+	if (item.href) {
+		await router.push(item.href);
+	}
+};
+</script>
+
+<template>
+	<div>
+		<router-link :to="cardUrl" @click="() => emit('folderOpened', { folder: props.data })">
+			<n8n-card :class="$style.card">
+				<template #prepend>
+					<n8n-icon
+						data-test-id="folder-card-icon"
+						:class="$style['folder-icon']"
+						icon="folder"
+						size="large"
+					/>
+				</template>
+				<template #header>
+					<n8n-heading tag="h2" bold size="small" data-test-id="folder-card-name">
+						{{ data.name }}
+					</n8n-heading>
+				</template>
+				<template #footer>
+					<div :class="$style['card-footer']">
+						<n8n-text
+							size="small"
+							color="text-light"
+							:class="$style['info-cell']"
+							data-test-id="folder-card-workflow-count"
+						>
+							{{ data.workflowCount }} {{ i18n.baseText('generic.workflows') }}
+						</n8n-text>
+						<n8n-text
+							size="small"
+							color="text-light"
+							:class="$style['info-cell']"
+							data-test-id="folder-card-last-updated"
+						>
+							{{ i18n.baseText('workerList.item.lastUpdated') }}
+							<TimeAgo :date="String(data.updatedAt)" />
+						</n8n-text>
+						<n8n-text
+							size="small"
+							color="text-light"
+							:class="$style['info-cell']"
+							data-test-id="folder-card-created"
+						>
+							{{ i18n.baseText('workflows.item.created') }}
+							<TimeAgo :date="String(data.createdAt)" />
+						</n8n-text>
+					</div>
+				</template>
+				<template #append>
+					<div :class="$style['card-actions']" @click.prevent>
+						<n8n-breadcrumbs
+							:items="breadCrumbsItems"
+							:path-truncated="true"
+							:show-border="true"
+							:highlight-last-item="false"
+							theme="small"
+							data-test-id="folder-card-breadcrumbs"
+							@item-selected="onBreadcrumbsItemClick"
+						>
+							<template v-if="data.homeProject" #prepend>
+								<div :class="$style['home-project']">
+									<n8n-link :to="`/projects/${data.homeProject.id}`">
+										<ProjectIcon :icon="projectIcon" :border-less="true" size="mini" />
+										<n8n-text size="small" :compact="true" :bold="true" color="text-base">
+											{{ projectName }}
+										</n8n-text>
+									</n8n-link>
+								</div>
+							</template>
+						</n8n-breadcrumbs>
+						<n8n-action-toggle
+							v-if="actions.length"
+							:actions="actions"
+							theme="dark"
+							data-test-id="folder-card-actions"
+							@action="onAction"
+						/>
+					</div>
+				</template>
+			</n8n-card>
+		</router-link>
+	</div>
+</template>
+
+<style lang="scss" module>
+.card {
+	transition: box-shadow 0.3s ease;
+	cursor: pointer;
+
+	&:hover {
+		box-shadow: 0 2px 8px rgba(#441c17, 0.1);
+	}
+}
+.folder-icon {
```

Comment:
if we stick to `camelCase`, then it's easier to reference in the template `$style.folderIcon` instead of `$style['folder-icon']`

Reply:
I know this is a personal preference but I would like to stick to kebab-case for css, even tho it's a bit more work to reference from Vue template

Reply:
Yes, this is optional nitpick. It's your call. Creates a bit of inconsistency in the codebase, but it's easier to read the css (and maybe the templates too)